src/exchanges/gemini-client.js

```diff
@@ -255,6 +255,18 @@ class GeminiClient extends EventEmitter {
 
     if (!market) return;
 
+    if (msg.type === "heartbeat") {
+      // if using sequenceId a heartbeat event will arrive which includes sequenceId.
+      // you'll need to receive heartbeat, otherwise sequence will have a gap in next l2update,
+      // so emit an l2update w/no ask or bid changes, only including the sequenceId
+      // ex: '{"type":"heartbeat","socket_sequence":272}'
+      const sequenceId = msg.socket_sequence;
```

Comment:
This will always emit an l2update regardless of whether the socket is subscribed.  We need to add a check on `subscription.level2updates` and only emit if the user has subscribed to `l2updates`.